AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1647,6 +1647,7 @@ newinform.com###tg-vk
 newinform.com##.tg-button
 livemint.com##div[class^="newslettersub"]
 livemint.com###lastAd
+timesofindia.com##.newsletter_hook_content
```

Comment:
I guess, it should be `m-timesofindia-com.cdn.ampproject.org` instead of `timesofindia.com` or `m-timesofindia-com.cdn.ampproject.org,timesofindia.com`.
The same for `timesofindia.com##.adspacing`